src/auditlog/models.py

```diff
@@ -94,6 +94,9 @@ def get_for_objects(self, queryset):
 
         if isinstance(primary_keys[0], integer_types):
             return self.filter(content_type=content_type).filter(Q(object_id__in=primary_keys)).distinct()
+        elif isinstance(queryset.model._meta.pk, models.UUIDField):
```

Comment:
Well, we need the value later on, so the query would have to be done anyways. Django should cache this, so it should be no problem. However, if you think it means having an extra query you can test this locally and supply some evidence, I would be interested in that.